spec/classes/database_postgresql_spec.rb

```diff
@@ -31,7 +31,7 @@
           end
 
           it { is_expected.to compile.with_all_deps }
-          it { is_expected.to contain_exec('update_pgpass').with_command('echo node01.example.com:5432:zabbix-server:zabbix-server:zabbix-server >> /root/.pgpass') }
+          it { is_expected.to contain_exec('update_pgpass').with_command('echo 'node01.example.com:5432:zabbix-server:zabbix-server:zabbix-server' >> /root/.pgpass') }
```

Comment:
This doesn't look right. Seems like there should be double quotation marks around the entire string, with single quotation marks within, like the lines below here.

Reply:
Thanks. I'm completely new to writing specs - so that was a quick attempt. I'll try your suggestion later today